docs/fuel-101/technical-roadmap.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Technical Roadmap
+category: Fuel 101
+---
+
+# Technical Roadmap
+
+## Testnet Roadmap
+
+#### New abi encoding scheme
+
+The new abi encoding scheme will enable support for nested heap types.
+This enables developers to return arbitrarily nested heap types from contracts and scripts, and log any arbitrary data type.
+A data type allocated on the heap can grow or shrink in size.
+Compared to the built-in array and tuple types which are allocated on the stack and cannot grow in serialize.
+
+#### Forkless upgrades in Fuel Core
+
+Supporting forkless upgrades to the fuel core will allow changing logic on the fly for all nodes in the network.
+This minimizes the impact to the network when upgrades take place.
+
+#### SRC-20 standardization for bridged assets
+
+#### Fuel wallet connectors
+
+Fuel wallet connectors enable users to sign transactions on the Fuel network using wallets such as Metamask or Phantom.
+These wallets are able to be used on Fuel through the use of predicates.
+
+#### The Graph integration
+
+Integrating Fuel with The Graph allows developers to easily index and query blockchain data for their Dapps.
```

Comment:
```suggestion
Integrating Fuel with The Graph allows developers to easily index and query blockchain data for their DApps.
```

Reply:
Can close after this discussion
https://fuellabs.slack.com/archives/C060N9LTYG7/p1712860720233399?thread_ts=1712767928.815259&cid=C060N9LTYG7